From 1b7c5651ab4d62429b3067f1aedac424f1821e1c Mon Sep 17 00:00:00 2001
From: Michael Scott <mike@foundries.io>
Date: Wed, 3 Aug 2022 15:39:44 -0700
Subject: [PATCH 322/322] [FIO internal] imx8mn_evk: use common
 spl_mmc_get_uboot_raw_sector()

This function moved to mach-imx/spl.c

Signed-off-by: Michael Scott <mike@foundries.io>
---
 board/freescale/imx8mn_evk/spl.c | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/board/freescale/imx8mn_evk/spl.c b/board/freescale/imx8mn_evk/spl.c
index 1c4f537a18a..530d437bd85 100644
--- a/board/freescale/imx8mn_evk/spl.c
+++ b/board/freescale/imx8mn_evk/spl.c
@@ -223,18 +223,3 @@ void board_init_f(ulong dummy)
 
 	board_init_r(NULL, 0);
 }
-#ifdef CONFIG_SPL_MMC_SUPPORT
-
-#define UBOOT_RAW_SECTOR_OFFSET 0x40
-unsigned long spl_mmc_get_uboot_raw_sector(struct mmc *mmc)
-{
-	u32 boot_dev = spl_boot_device();
-	switch (boot_dev) {
-		case BOOT_DEVICE_MMC1:
-			return CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR;
-		case BOOT_DEVICE_MMC2:
-			return CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR - UBOOT_RAW_SECTOR_OFFSET;
-	}
-	return CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR;
-}
-#endif
-- 
2.34.1

